Dockerfile

```
@@ -1,17 +1,10 @@
-FROM openjdk:17-slim
-CMD ["/usr/local/openjdk-17/bin/java", "-jar", "/opt/ssdc-rm-job-processor.jar"]
+FROM eclipse-temurin:17-jdk-alpine
 
+ARG JAR_FIL=ssdc-rm-job-processo*.jar
```

```suggestion
ARG JAR_FILE=ssdc-rm-job-processor*.jar
```
Some typos braking this?